<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PollenDetect</class>
 <widget class="QWidget" name="PollenDetect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1799</width>
    <height>915</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_6">
   <item>
    <widget class="QScrollArea" name="scrollArea_5">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1762</width>
        <height>1060</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>1660</width>
        <height>1060</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QWidget" name="widget" native="true">
         <widget class="QPushButton" name="btn_src2gray">
          <property name="geometry">
           <rect>
            <x>530</x>
            <y>220</y>
            <width>41</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>=&gt;</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_open">
          <property name="geometry">
           <rect>
            <x>1240</x>
            <y>530</y>
            <width>91</width>
            <height>81</height>
           </rect>
          </property>
          <property name="text">
           <string>open</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>1160</x>
            <y>40</y>
            <width>54</width>
            <height>12</height>
           </rect>
          </property>
          <property name="text">
           <string>二值化图</string>
          </property>
         </widget>
         <widget class="QScrollArea" name="scrollArea_2">
          <property name="geometry">
           <rect>
            <x>590</x>
            <y>70</y>
            <width>471</width>
            <height>321</height>
           </rect>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>469</width>
             <height>319</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_imgGray">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QPushButton" name="btn_gray">
          <property name="geometry">
           <rect>
            <x>590</x>
            <y>390</y>
            <width>31</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>show</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_src">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>390</y>
            <width>31</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>show</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>50</y>
            <width>54</width>
            <height>12</height>
           </rect>
          </property>
          <property name="text">
           <string>原图</string>
          </property>
         </widget>
         <widget class="QScrollArea" name="scrollArea_3">
          <property name="geometry">
           <rect>
            <x>1170</x>
            <y>60</y>
            <width>471</width>
            <height>321</height>
           </rect>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_3">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>469</width>
             <height>319</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_imgBinary">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>590</x>
            <y>50</y>
            <width>54</width>
            <height>12</height>
           </rect>
          </property>
          <property name="text">
           <string>灰度图</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_gray2bianry">
          <property name="geometry">
           <rect>
            <x>1100</x>
            <y>220</y>
            <width>41</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>=&gt;</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_binary">
          <property name="geometry">
           <rect>
            <x>1170</x>
            <y>380</y>
            <width>31</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>show</string>
          </property>
         </widget>
         <widget class="QSpinBox" name="spinBox">
          <property name="geometry">
           <rect>
            <x>1100</x>
            <y>190</y>
            <width>42</width>
            <height>22</height>
           </rect>
          </property>
          <property name="maximum">
           <number>256</number>
          </property>
         </widget>
         <widget class="QScrollArea" name="scrollArea">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>70</y>
            <width>431</width>
            <height>321</height>
           </rect>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>429</width>
             <height>319</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_imgSrc">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QScrollArea" name="scrollArea_4">
          <property name="geometry">
           <rect>
            <x>630</x>
            <y>480</y>
            <width>541</width>
            <height>351</height>
           </rect>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_4">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>539</width>
             <height>349</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_imgCircles">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>630</x>
            <y>460</y>
            <width>81</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>霍夫圆变化图</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_Circles">
          <property name="geometry">
           <rect>
            <x>630</x>
            <y>830</y>
            <width>31</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>show</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_binary2circles">
          <property name="geometry">
           <rect>
            <x>550</x>
            <y>580</y>
            <width>41</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>=&gt;</string>
          </property>
         </widget>
         <widget class="QSlider" name="verticalSlider">
          <property name="geometry">
           <rect>
            <x>1110</x>
            <y>290</y>
            <width>22</width>
            <height>160</height>
           </rect>
          </property>
          <property name="maximum">
           <number>256</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
         <widget class="QSpinBox" name="spinBox_min">
          <property name="geometry">
           <rect>
            <x>580</x>
            <y>520</y>
            <width>42</width>
            <height>22</height>
           </rect>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
         <widget class="QSpinBox" name="spinBox_max">
          <property name="geometry">
           <rect>
            <x>580</x>
            <y>640</y>
            <width>42</width>
            <height>22</height>
           </rect>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="value">
           <number>900</number>
          </property>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>580</x>
            <y>500</y>
            <width>31</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>min</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>590</x>
            <y>620</y>
            <width>41</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>max</string>
          </property>
         </widget>
         <widget class="QScrollArea" name="scrollArea_6">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>430</y>
            <width>431</width>
            <height>321</height>
           </rect>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_6">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>429</width>
             <height>319</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_Canny">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QPushButton" name="btn_canny">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>580</y>
            <width>41</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>=&gt;</string>
          </property>
         </widget>
         <widget class="QSlider" name="verticalSlider_cannyMin">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>630</y>
            <width>22</width>
            <height>160</height>
           </rect>
          </property>
          <property name="maximum">
           <number>256</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
         <widget class="QSlider" name="verticalSlider_cannyMax">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>630</y>
            <width>22</width>
            <height>160</height>
           </rect>
          </property>
          <property name="maximum">
           <number>256</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton">
          <property name="geometry">
           <rect>
            <x>1170</x>
            <y>780</y>
            <width>41</width>
            <height>51</height>
           </rect>
          </property>
          <property name="text">
           <string>save</string>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>800</y>
            <width>21</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>min</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_8">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>800</y>
            <width>21</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>max</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_2">
          <property name="geometry">
           <rect>
            <x>1250</x>
            <y>690</y>
            <width>111</width>
            <height>81</height>
           </rect>
          </property>
          <property name="text">
           <string>开启线程1</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_3">
          <property name="geometry">
           <rect>
            <x>1360</x>
            <y>690</y>
            <width>111</width>
            <height>81</height>
           </rect>
          </property>
          <property name="text">
           <string>关闭线程1</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
